website/src/docs/aws-s3.md

```diff
@@ -152,7 +152,7 @@ uppy.use(AwsS3, {
       },
       body: JSON.stringify({
         filename: file.name,
-        contentType: file.type
+        type: file.type
```

Comment:
This example belongs to this PHP file: https://github.com/transloadit/uppy/blob/master/examples/aws-presigned-url/s3-sign.php, which uses `contentType`.